### PR TITLE
Fix the problem with several singlequotes in fixture and test names (fix #754)

### LINKED ISSUE #754: Test get stuck if `testName` contains more than one singlequote
Test to reproduce:
```js
fixture `Simple demo`
    .page `http://example.com`;

test("one'two'three", async t => {
    await t.click('body');
});
```
We will get this on the client:

### PR DESCRIPTION
**Pull request by AlexanderMoskovkin**
/cc @inikulin @VasilyStrelyaev

### CODE REVIEW

File: test/functional/fixtures/regression/gh-754/testcafe-fixtures/index-test.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+fixture `fixture'with'serveral'singlequotes`;
```
Comment (VasilyStrelyaev) [inline test/functional/fixtures/regression/gh-754/testcafe-fixtures/index-test.js]: > serveral

**Comment by VasilyStrelyaev** [comment]
\r-

File: test/functional/fixtures/regression/gh-754/testcafe-fixtures/index-test.js (hunk 1 above)
Comment (VasilyStrelyaev) [inline test/functional/fixtures/regression/gh-754/testcafe-fixtures/index-test.js]: and test name

**Comment by AlexanderMoskovkin** [comment]
FPR

**Comment by inikulin** [comment]
lgtm

**Comment by VasilyStrelyaev** [comment]
lgtm

**Pull request merged**